build/Dockerfile

```diff
@@ -1,27 +1,30 @@
-FROM golang:1.21 AS go_builder
+FROM --platform=${BUILDPLATFORM} golang:1.21 AS go_builder
```

Comment:
I am not sure that it will work in the case of multi-arch builds like  `--platform linux/amd64,linux/arm64`.